content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Example of MapReduce With Ezmeral Data Fabric(MapR) Database Binary Table
+date: 2022-11-14T15:56:03.596Z
+featuredBlog: false
+author: Raymond Yan
+authorimage: /img/raymondyan-profile.png
+thumbnailimage: /img/mapreduce_with_edf_binarytable.png
+disable: false
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapReduce
+---
+
+## Introduction
+
+
+
+This article will cover:
+
+
+
+1. How to create Development Environment for HPE Ezmeral Data Fabric on Linux, Windows and Mac.
+
+
+

```

Comment:
For a better rendering of the 2 next sentences, I would suggest to move them left.